commit 9cea138fe0abc09c100772130560a19f72ce7a3d
Author: Tim Gover <tim.gover@raspberrypi.org>
Date:   Tue Oct 16 11:41:30 2018 +0100

    ASoC: Fix compatible string for Allo Piano DAC
    
    The RPi PCM512x had the wrong compatible for the Allo Piano DAC which it
    replaced. Change this to the correct value.

diff --git a/sound/soc/bcm/rpi-pcm512x-soundcard.c b/sound/soc/bcm/rpi-pcm512x-soundcard.c
index 66289208f36e..75201421e358 100644
--- a/sound/soc/bcm/rpi-pcm512x-soundcard.c
+++ b/sound/soc/bcm/rpi-pcm512x-soundcard.c
@@ -255,7 +255,7 @@ static struct snd_rpi_pcm512x_drvdata drvdata_justboom_dac = {
 };
 
 static const struct of_device_id snd_rpi_pcm512x_of_match[] = {
-	{ .compatible = "allo,allo-piano-dac",
+	{ .compatible = "allo,piano-dac",
 		.data = (void *) &drvdata_allo_piano_dac },
 	{ .compatible = "dionaudio,dionaudio-loco-v2",
 		.data = (void *) &drvdata_dionaudio_loco_v2 },
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1212_9cea138fe0abc09c100772130560a19f72ce7a3d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1212_9cea138fe0abc09c100772130560a19f72ce7a3d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
